tpgtools/resource.go

```diff
@@ -722,9 +726,10 @@ func (r *Resource) loadDCLSamples() []Sample {
 
 		primaryResource := *sample.PrimaryResource
 		parts := strings.Split(primaryResource, ".")
-		primaryResourceName := parts[len(parts)-2]
+		primaryResourceName := snakeToTitleCase(parts[len(parts)-2])
 
 		if !versionMatch || primaryResourceName != resourceType {
+			glog.Errorf("skipping %s since no match with %s.", primaryResourceName, resourceType)
 			continue
 		}
```

Comment:
```suggestion
		if !versionMatch ||  {
			continue
		} else if primaryResourceName != resourceType {
			glog.Errorf("skipping %s since no match with %s.", primaryResourceName, resourceType)
			continue
		}
```